src/content/guidelines/motion/basics.mdx

```diff
@@ -85,9 +83,9 @@ Productive motion creates a sense of efficiency and responsiveness, while bein
 
 Expressive motion delivers enthusiastic and vibrant, thus more visible movement. Use expressive motion for significant moments such as user opening a new page or clicking the primary action button, or when the movement itself conveys a meaning or needs to be noticed by the user such as system alert or notification box appearing.
 
-Productivity and Expression are both essential to an interface. Reserve Expressive motion for occasional, important moments to better capture user’s attention, and offer rhythmic break to the productive experience.
+Productivity and Expression are both essential to an interface. Reserve Expressive motion for occasional, important moments to better capture the user’s attention and offer a rhythmic break to the productive experience..
 
-Our [components](../../components/overview/) have basic motion build in. Use this guidance to customize motion, combine and coordinate motion through out experiences.
+[Carbon components](../../components/overview/) have motion built in for microinteractions. However, the motion design of the overarching UI — that is, how the components interact with each other and enter and exit the page itself — is up to each product’s design team to implement. Use this guidance to customize, combine, coordinate, and choreograph this aspect of motion in the UI.
```

Comment:
I would move this paragraph under the parent paragraph of Style. It pertains to both expressive and productive. Right now it reads like its only for expressive.

Reply:
Hmm I tried but it feels like it's disrupting the flow of explaining what Style is. Maybe it can be a intro paragraph even before Style? @aagonzales